.travis.yml

```diff
@@ -7,5 +7,5 @@ before_install:
     - git submodule update --init --recursive
 language: php
 php:
-  - 7.1
+  - 7.4
```

Comment:
We're not all the way to 7.3 yet..
```suggestion
  - 7.3
  - 7.4
```

Reply:
done

Reply:
Why not both?

Reply:
Misread your suggestion. I was trying to look up if it was possible to specify both but the SSL was failing on their docs so I just went with one.